src/gdcmodels/esmodels/segment_cnv_centric/mapping.yaml

```diff
@@ -0,0 +1,606 @@
+---
+_size:
+  enabled: true
+dynamic: strict
+properties:
+  chromosome:
```

Comment:
The CNV fields for `cnv_occurrence_centric` are nested under a `cnv` property whereas in this mapping they appear at the root of the document. I just want to verify this is the desired shape.

Reply:
same comment as above

Reply:
Thank you for clarifying